test/jdk/javax/swing/SwingWorker/TestDoneBeforeDoInBackground.java

```diff
@@ -83,25 +86,27 @@ public void propertyChange(PropertyChangeEvent evt) {
                     // Before doInBackground method is invoked,
                     // SwingWorker notifies PropertyChangeListeners about the
                     // property change to StateValue.STARTED
-                    if (doInBackgroundFinished.get() &&
-                         worker.getState() == SwingWorker.StateValue.STARTED) {
+                    if (doInBackgroundFinished.get()
+                        && worker.getState() == SwingWorker.StateValue.STARTED) {
                         throw new RuntimeException(
                                "PropertyChangeListeners called with " +
                                "state STARTED after doInBackground is invoked");
                     }
+
                     // Ensure the STARTED state is notified
                     // before doInBackground started running
-                    if (doInBackgroundStarted.get() &&
-                         worker.getState() == SwingWorker.StateValue.STARTED) {
+                    if (doInBackgroundStarted.get()
+                        && worker.getState() == SwingWorker.StateValue.STARTED) {
                         throw new RuntimeException(
                                "PropertyChangeListeners called with " +
                                "state STARTED before doInBackground is finished");
                     }
+
                     // After the doInBackground method is finished
                     // SwingWorker notifies PropertyChangeListeners
                     // property change to StateValue.DONE
-                    if (doInBackgroundFinished.get() &&
-                            worker.getState() != SwingWorker.StateValue.DONE) {
+                    if (worker.getState() != SwingWorker.StateValue.DONE
+                          && doInBackgroundFinished.get()) {
```

Comment:
```suggestion
                    if (worker.getState() != SwingWorker.StateValue.DONE
                        && doInBackgroundFinished.get()) {
```
This should also use four-space indentation on the continuation line for consistency with other cases.